requirements/base.in

```diff
@@ -1,2 +1 @@
-requests
```

Comment:
I chose not to use OpenAI library so I don't have to keep up with the changes. In my experience, API client libraries break compatibility more often than the API they call.
Another consideration is that client libraries tend to provide little value while imposing their own constraints, like the name of the environment variable for the token. Other common things (that don't apply in this case, JFYI) include the inability to set timeouts and proxies, use custom logging and metrics integrations.
My bet didn't pan out as OpenAI seems to have deprecated the v1 API this tool uses.
I want you to consider switching to using the API directly (through requests or httpx), but I leave this decision up to you.